tests/rados/monitor_configurations.py

```diff
@@ -292,6 +296,14 @@ def verify_set_config(self, **kwargs):
                 entry["name"].lower() == kwargs["name"].lower()
                 and entry["section"].lower() == kwargs["section"].lower()
             ):
+                # temporary fix, should be changed later, current logic is not robust
```

Comment:
please add a backlog task to rewrite the method

Reply:
Task created: https://issues.redhat.com/browse/RHCEPHQE-15039

Reply:
Thanks @pdhiran